src/lib/components/tablev2/Tablev2.component.js

```diff
@@ -119,4 +145,6 @@ function Table({
 }
 
 Table.SingleSelectableContent = SingleSelectableContent;
+Table.Search = Search;
+Table.SearcWithQueryParams = SearchWithQueryParams;
```

Comment:
I think there is a typo here:
```suggestion
Table.SearchWithQueryParams = SearchWithQueryParams;
```